Cargo.toml

```diff
@@ -12,11 +12,15 @@ ed25519-dalek = "1"
 prost = "0.7"
 serde = { version = "1", features = ["serde_derive"] }
 subtle-encoding = { version = "0.5", features = ["bech32-preview"] }
+nix = "0.17"
 tendermint = { version = "0.19" }
 tendermint-proto = "0.19"
 tendermint-p2p = { version = "0.19" }
 thiserror = "1"
 tracing = "0.1"
+tracing-core = "0.1"
+tracing-subscriber = "0.2"
+vsock = "0.2"
```

Comment:
is there a need to move all these dependencies to the top-level crate? They are only related to Nitro -- so can just be in the Nitro crates? SGX build currently fails because of that: https://github.com/crypto-com/tmkms-light/pull/68/checks?check_run_id=2564304417#step:4:304

Reply:
for the tracing stuff, it can be in the helper's shared.rs module?